docs/templates/edit-example-link.html

```diff
@@ -0,0 +1,10 @@
+<div class="edit-example-link">
+    {% if pagename.split('/')[-1].startswith('plot_') %}
+        <a href="https://github.com/fairlearn/fairlearn/edit/main/examples/{{ pagename.split('/')[1]}}.py">
+            Edit this example on GitHub
+        </a>
+        <a href="https://github.com/fairlearn/fairlearn/issues/new?assignees=&labels=Documentation&template=doc_improvement.md&title=DOC%20Issue%20in%20{{ pagename.split('/')[1] }}&body=https://github.com/fairlearn/fairlearn/edit/main/examples/{{ pagename.split('/')[1] }}.py">
```

Comment:
Currently, the template isn't used because the body is specified explicitly. I personally prefer having the template over the exact link because it encourages people to write something meaningful (and having the title point to the example name seems sufficient). We also need to use lowercase `documentation`, otherwise it doesn't work. So i'd suggest something like this:
 
```suggestion
        <a href="https://github.com/fairlearn/fairlearn/issues/new?labels=documentation&template=doc_improvement.md&title=DOC%20Issue%20in%20Example%20Notebook%20{{ pagename.split('/')[1] }}.py>
```

Which should result in a link like this: https://github.com/fairlearn/fairlearn/issues/new?labels=documentation&template=doc_improvement.md&title=DOC%20Issue%20in%20Example%20Notebook%20plot_quickstart_counts.py